drivers/client_access_control/app/views/client_access_control/clients/show.haml

```diff
@@ -6,13 +6,10 @@
   .o-page__header
     = render 'clients/aliases'
   = render 'clients/tab_navigation', current: client_path(@client)
-  - if can_access_some_version_of_clients? && ! GrdaWarehouse::Config.get(:multi_coc_installation)
-    %section.o-dashboard__block--primary.client__summary
-      = render 'client_access_control/clients/client_summary'
-  - if can_view_full_client_dashboard?
-    = render 'client_access_control/clients/rollups'
-  - elsif can_view_limited_client_dashboard?
-    = render 'client_access_control/clients/rollups_limited'
+  - if GrdaWarehouse::Config.get(:client_dashboard).to_sym == :va
```

Comment:
I think this is fine for now, but if we scale this beyond 2 simple dashboards, it might be a good idea to create dashboard classes with the version specific logic in there, and then `render @dashboard`

Reply:
Maybe I should just change it to `render GrdaWarehouse::Config.get(:client_dashboard)` for now.  No `if` necessary.  Pretty sure that will do the same thing since it should return `va` or `default` currently.